File: src/main/java/ch/hevs/sdi/simpleapp/FancyPrinter.java
package ch.hevs.sdi.simpleapp;

import org.apache.commons.logging.Log;
import org.apache.commons.logging.LogFactory;

public class FancyPrinter implements Printer {
    private Log log = LogFactory.getLog(FancyPrinter.class);

    @Override
    public void print(String message) {
        log.info(message);
    }
}

File: src/main/java/ch/hevs/sdi/simpleapp/FancyPrinter.kt
package ch.hevs.sdi.simpleapp

import org.apache.commons.logging.LogFactory

class FancyPrinter: Printer {
    private val log = LogFactory.getLog(FancyPrinter::class.java)

    override fun print(message: String) = log.info(message)
}
